### DISCUSSION TITLE
Can't remove speed and play button once applied scientific data to mesh

### DISCUSSION THREAD

**Discussion by tusharsable** [iTwin/itwinjs-core#2940] [Q&A]
Hi All, 

I am unable to remove the play button and slider with auxillary data from itwin viewer. Can anyone suggest me a code to do so , or disable it. we have used auxiliary data and showed colors on a mesh 
![image](https://user-images.githubusercontent.com/66006244/146573671-c960b3e6-811f-40ab-9e09-6272f75b5369.png)

**Comment by pmconne** [comment]
@bsteinbk

**Reply by NancyMcCallB** [reply]
We don't currently have an API to hide and show the DefaultViewOverlay in a Viewport -- it's automatically displayed if animation info is detected. We'll add a show/hide API. In the short term, you can brute-force it by querying the DOM for the container (document.querySelector("div.uifw-view-overlay") ) and setting the display to "none". If you want to show it after hiding it, set the display to "flex".
